waterlog/src/Components/Usage.js

```diff
@@ -45,6 +45,7 @@ class Usage extends Component {
       return <div>
         <p><b>Cost</b></p>
         <DailyCostsReports props={this.props.dailyCost} />
+        <br/>
```

Comment:
Rather use styling to space the components.

Reply:
I added style.